grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -733,4 +733,24 @@ class NotifyService implements SendgridSupport, Status {
                 )
         )
     }
+
+    Map<Boolean, String> sendAddedCGToProjectNotification(String consentKey, String projectKey) {
+        Map<String, String> values = new HashMap<>()
+        Issue consent = Issue.findByProjectKey(consentKey)
+        Issue project = Issue.findByProjectKey(projectKey)
+        values.put("projectLink", getShowIssueLink(project))
```

Comment:
TOL (thinking out loud): It would be nice if we didn't have to manually add these strings every time to the map that feeds `NotifyArguments`. I suggest we write a tech-debt story to standardize the `values` object that `NotifyArguments` takes so we can use standard enums.